doc/README.md

```diff
@@ -1,14 +1,20 @@
-# Cloud iNfrastructure Telco Task Force (CNTT)
+# Anuket Specifications
 
 ## Table of Contents
+
 * [Governance & Community Guidelines](gov/README.md)
```

Comment:
should all of these references be changed to rst versions?

Reply:
Relative links are corrected by the relative link collector plugin. We can change them to 'proper' rst linking once all document conversions pr-s are merged. 
But we should not have any md files at all.

Reply:
Fixed in a75a857